config/sewing-kit/index.ts

```diff
@@ -2,42 +2,37 @@ import {
   Package,
   createComposedProjectPlugin,
   createProjectTestPlugin,
-  createProjectBuildPlugin,
-} from '@sewing-kit/plugins';
-import {react} from '@sewing-kit/plugin-react';
-import {javascript, updateBabelPreset} from '@sewing-kit/plugin-javascript';
-import {typescript} from '@sewing-kit/plugin-typescript';
+} from '@sewing-kit/core';
+import {babel} from '@sewing-kit/plugin-babel';
 import {packageBuild} from '@sewing-kit/plugin-package-build';
 import {} from '@sewing-kit/plugin-jest';
 
-import {addLegacyDecoratorSupport} from './plugin';
-
 export function quiltPackage({
   jestEnv = 'jsdom',
-  useReact = false,
   jestTestRunner = 'jest-circus',
+  polyfill = true,
 } = {}) {
   return createComposedProjectPlugin<Package>('Quilt.Package', [
-    javascript(),
-    typescript(),
-    useReact && react(),
+    babel({
+      config: {
+        presets: [
+          [
+            '@shopify/babel-preset',
+            {
+              typescript: true,
+              react: true,
+              // The preset polyfills by default, we need explicit options to
+              // disable that
+              ...(polyfill ? {} : {useBuiltIns: false, corejs: false}),
+            },
+          ],
+        ],
+      },
+    }),
     packageBuild({
       nodeTargets: 'node 12.14.0',
       browserTargets: 'extends @shopify/browserslist-config',
     }),
-    createProjectBuildPlugin('Quilt.PackageBuild', ({hooks}) => {
```

Comment:
babelIgnorePatterns no longer exists, and we don't need to add legacy decorator support because `@shopify/babel-preset` already enables it correctly